packages/starknet_flutter/lib/src/views/wallet/routes/restore_wallet/protect_wallet_screen.dart

```diff
@@ -19,21 +20,32 @@ class ProtectWalletScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Padding(
-      padding: const EdgeInsets.all(16.0),
+      padding: const EdgeInsets.all(20.0),
       child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
         children: [
-          const Text(
+          Text(
             "This extra layer of security helps prevent someone with your phone from accessing your funds.",
+            style: GoogleFonts.poppins(
+              fontWeight: FontWeight.w400,
+            ),
           ),
-          const SizedBox(height: 32),
+          const SizedBox(height: 10),
+          Text(
```

Comment:
Maybe put a TODO so we don't forget to remove the text when we can do it in one prompt